data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+. /usr/local/bin/release-image.sh
+
+wait_for_existence() {
+    while [ ! -e "${1}" ]
+    do
+        sleep 5
+        echo "bootkube not done, sleeping.."
+    done
+}
+
+# Wait for the API to become active
+until curl http://localhost:6385/v1/nodes; do
+    echo Waiting for API
+    sleep 5
+done
+
+# Wait for the nodes to become active after introspection.
+while curl http://localhost:6385/v1/nodes | jq '.nodes[] | .provision_state' | grep -v active; do
+    echo Waiting for nodes to be come active
+    sleep 5
+done
+
+# Just for debugging for now.
+for node in $(curl http://localhost:6385/v1/nodes | jq -r '.nodes[] | .uuid'); do
+    echo host $node
+    curl http://localhost:5050/v1/introspection/$node/data
+done
+
+# Wait for bootkube to come up.  We want to talk to the full instance not
+# the bootstrap openshift
+wait_for_existence /opt/openshift/.bootkube.done
+
+echo nodes are up
+
+BAREMETAL_OPERATOR_IMAGE=$(image_for baremetal-operator)
+
+# Now we go through each image..
+for node in $(curl http://localhost:6385/v1/nodes | jq -r '.nodes[] | .uuid'); do
+    echo host $node
+    # And use the baremetal operator tool to load the introspection data into
+    # the BareMetalHost CRs as annotations, which BMO then picks up.
+    podman run --quiet --net=host \
+        --rm \
+        --name baremetal-operator \
+        --entrypoint /bin/make-introspection \
+        "${BAREMETAL_OPERATOR_IMAGE}" \
+        http://localhost:5050/v1 $node
```

Comment:
This command will just give us the JSON output.  I think we'll need to find which BareMetalHost each ironic node belongs to, and then use `oc patch` to set the status annotation.

Reply:
ok.  Going to fire it up and try it live.